client/state/ui/editor/post/actions.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:ui:editor:post:actions' );
+
+export function setEditingMode( mode, modeTitle, site ) {
+	debug( 'setEditingMode', mode, modeTitle, site );
+	return { type: 'TODO' };
+}
+
+export function startEditingNew( site, postOptions ) {
+	debug( 'startEditingNew', site, postOptions );
+	return { type: 'TODO' };
+}
+
+export function startEditingExisting( site, postID ) {
+	debug( 'startEditingExisting', site, postID );
+	return { type: 'TODO' };
+}
+
+export function toggleStickyStatus( currentStatus ) {
```

Comment:
As a matter of fact, there was with @mtias : https://github.com/Automattic/wp-calypso/pull/3025#discussion_r51764414

Arguments pro-individual-property:
- Most of [this](https://github.com/Automattic/wp-calypso/pull/3025/files#diff-96fbb5e8d8b2d3c5308a18e3b9b20161R59), [this](https://github.com/Automattic/wp-calypso/pull/3025/files#diff-c65fc9514aaa064f1fa8671d6b735e28R51) and similar function logic should be ported from component to actions and reducers. Using proper abstraction allows us to do so in the future.
- In the actions file we can underneath use the old approach, [like in notices](https://github.com/Automattic/wp-calypso/blob/master/client/state/notices/actions.js#L43)
- Component to be logically pure should import only the action it needs instead of one mega-setter
- It allows us to restructure underlying data object only by manipulating those setters
- We have to have actionCreators either way. Having individual ones has no downside